docs/fr/index.md

```diff
@@ -1,153 +1,143 @@
-# Démarrer sur l'espace de travail d'analyse avancée
+# Bienvenue dans l'espace de travail Advanced Analytics
```

Comment:
Change  "l'espace de travail Advanced Analytics" to l'espace de travail d'analyse avancée